app/views/devise/mailer/confirmation_instructions.html.erb

```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```

Comment:
You're not using `confirmable` so remove this unused view.

Reply:
I'll remove this for now... but I defo need to put it back in, required for GDPR - the new marking laws think that says all online services must have double opt in for you to be able to email them... I'll add this as a new story :)